source/js/viewer-core.js

```diff
@@ -1011,7 +1011,7 @@ export default class ViewerCore
 
     initPlugins ()
     {
-        if (!this.settings.hasOwnProperty.call('plugins'))
+        if (!this.settings.plugins)
```

Comment:
I would go with `!this.settings.hasOwnProperty('plugins')`. The difference is that your returns `undefined` if it doesn't exist, but `hasOwnProperty` returns a boolean. 

The difference is largely academic, but if you get in the habit of checking for the existence of a property by looking for it with `hasOwnProperty` then you will avoid some bugs later on.